experimenter/experimenter/nimbus-ui/src/styles/index.scss

```diff
@@ -135,6 +135,26 @@ select.form-control.is-warning {
   height: 18px;
 }
 
+.copy-to-clipboard-icon {
+  min-width: 18px;
+  width: 18px;
+  height: 18px;
+}
+
+.copy-button:hover svg {
+  transform: scale(1.2);
+}
+
+.copy-button{
+  border: none;
+  margin-bottom: 0.35rem !important;
+}
+
+
+.copy-button:focus {
+  box-shadow: none !important;
+}
+
```

Comment:
oh, you can use bootstrap classes instead of having custom CSS, here are the docs https://react-bootstrap-v4.netlify.app/components/alerts/